Model/Cron.php

```diff
@@ -445,7 +445,7 @@ private function shouldSkipProcessingNotification($notification)
                 $authorisationSuccessFalseMinDate = new \DateTime('-10 minutes');
                 $createdAt = \DateTime::createFromFormat('Y-m-d H:i:s', $notification['created_at']);
 
-                $minutesUntilProcessing = $createdAt->diff($authorisationSuccessFalseMinDate)->i;
+                $minutesUntilProcessing = ($createdAt->format('U') - $authorisationSuccessFalseMinDate->format('U')) / 60;
```

Comment:
Why adding this change here?

Reply:
Yep, found this issue during testing for Blik, is in the desc:
>Unhappy flows for Blik send a AUTHORISATION success=false notification that was being ignored in some scenarios during cron run, same as #1014